app/controllers/posts_controller.rb

```diff
@@ -5,5 +5,56 @@ def index
 
   def show
     @post = Post.find_by(id: params.require(:post_id))
+    @user_id = params.require(:user_id)
+    @comment = Comment.new
+  end
+
+  def new
+    @post = Post.new
+    @user_id = current_user.id
+  end
+
+  def create
+    user = current_user
+    @post = Post.new(
+      title: post_params[:title],
+      text: post_params[:text],
+      user:,
+      comments_counter: 0,
+      likes_counter: 0
+    )
+
+    if @post.save
+      @post.increment_posts_counter
+      redirect_to "/users/#{user.id}/posts"
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def create_comment
```

Comment:
- [ ] Great job working on this milestone. To create a comment on behalf of the `current_user`, you need to have a `comments_controller` just as you had for `posts_controller`
`A comment is created on behalf of the currently logged in user (current_user) of the app.
The comments controller should have new and create actions`